frontend/src/pages/spotnet/dashboard/withdraw-all/withdraw_all.css

```diff
@@ -0,0 +1,198 @@
+body {
+    font-family:
+      'Rethink Sans',
+      -apple-system,
+      Roboto,
+      Helvetica,
+      sans-serif;
+  }
```

Comment:
remove this, I belive we already have font-family in globals.css

Reply:
Updated!